cypress/integration/BonnieFHIR/WebInterface/Patient/AttributeUI/TimingWidget.spec.js

```diff
@@ -0,0 +1,127 @@
+import * as helper from '../../../../../support/helpers'
+import * as bonnieLogin from '../../../../../support/BonnieFHIR/BonnieLoginLogout'
+import * as measureDetailsPage from '../../../../../pom/BonnieFHIR/WI/MeasureDetailsPage'
+import * as testPatientPage from '../../../../../pom/BonnieFHIR/WI/TestPatientPage'
+import * as bonnieUploadMeasure from '../../../../../support/BonnieFHIR/BonnieUploadMeasure'
+
+const measureName = 'HEDISBCS2020'
+const measureFileToUpload = 'HEDISBCS2020-v0-0-010-FHIR-4-0-1.zip'
+
+const lastNameSuffix = new Date().getTime()
+const distinctLastName = 'President' + lastNameSuffix
+const year = new Date().getFullYear() -1
+const date = new Date().getMonthFormatted().toString() + '/' + new Date().getDayFormatted().toString() + '/'
+  + year.toString()
+
+describe('Attribute UI: timing: Timing', () => {
+
+  before('Login', () => {
+
+    bonnieLogin.login()
+    bonnieUploadMeasure.UploadMeasureToBonnie(measureFileToUpload, false)
+    measureDetailsPage.navigateToMeasureDetails(measureName)
+
+  })
+  after('Log Out', () => {
+
+    bonnieLogin.logout()
+
+  })
+
+  it('Verify the Timing Widget', () => {
+
+    cy.get(measureDetailsPage.patientListing).then((patientListing) => {
```

Comment:
Not critical, but ideally it should be cy.get(measureDetailsPage.patientFraction).then((patientListing)

**patientFraction** has the total number of patients, but **patientListing** is just a number of passed/successful patients. It can be different if expected values are entered.